streamer/test/unit/streamer/test_config.py

```diff
@@ -1,4 +1,4 @@
-from unittest import TestCase
+from unittest2 import TestCase
```

Comment:
Ah, I looked around in there, but didn't noticed it because I looked for the import at the top. Thanks!